homeassistant/components/matter/diagnostics.py

```diff
@@ -0,0 +1,76 @@
+"""Provide diagnostics for Matter."""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Any
+
+from matter_server.common.helpers.util import dataclass_to_dict
+
+from homeassistant.components.diagnostics import REDACTED
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN
+from .helpers import get_device_id, get_matter
+
+ATTRIBUTES_TO_REDACT = {"chip.clusters.Objects.Basic.Attributes.Location"}
+
+
+def redact_matter_attribute(node_data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
shouldn't this be 'redact_matter_attributes' as it is processing multiple attributes ?